### DISCUSSION TITLE
Smiley Player Trader Release 1.6.0

### DISCUSSION THREAD

**Discussion by Mrcomputer1** [Mrcomputer1/SmileyPlayerTrader#44] [Releases]
# Change Log
* 1.19.3 Support
* It is now possible to configure who the auto thanks message is sent to.
* You can now use `/spt all` command to see all available trades on the server (permission for this command is `smileyplayertrader.alltradeslist`).
* It is now possible to set a limit to the amount of times an item can be purchased.
* Russian Translation (By @Fosyaaa on GitHub).
* Improved the Edit Product GUI.
* Code Improvement: Cleaned up the code for the GUI system.

**Note: We recommend backing up your SPT database and configuration file, just in case. This version makes changes to the structure of the SPT database, if something unexpected, such as a power outage or server crash occurs, the database could be left in an inconsistent or corrupted state.**

# Updating Config
_Don't forget to make a backup of your config!_
**Option 1:** Look at [config.yml](src/main/resources/config.yml) and set the settings you want in your config.yml.
**Option 2:** Rename your config.yml and restart/reload your server and then re-apply your settings to the new config.yml and restart/reload the server again to apply them.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrcomputer1/SmileyPlayerTrader/releases/tag/v1.6.0'>Smiley Player Trader Release 1.6.0</a>.</em>

**Comment by TheRealFosya** [comment]
[ru_ru.zip](https://github.com/Mrcomputer1/SmileyPlayerTrader/files/10199341/ru_ru.zip)
That's it, Russian translation for new version.

**Reply by Mrcomputer1** [reply]
Thanks. I will release an update with the updated translation in a few days.

**Reply by Mrcomputer1** [reply]
In that file, `&aSelected` isn't translated.

**Reply by TheRealFosya** [reply]
[ru_ru.zip](https://github.com/Mrcomputer1/SmileyPlayerTrader/files/10225282/ru_ru.zip)
Now it's okay, i have a bunch versions of one file, i rewrited it.

**Reply by Mrcomputer1** [reply]
Thanks. I will release an update with the updated translation soon.
